frontend/src/app/pages/marker-page/marker-page.component.html

```diff
@@ -1,6 +1,48 @@
 <app-nav-bar>
     <div class="marker-nav">
         <div class="marker-nav--actions">
+            <button mat-mini-fab color="primary" class="window-change-button" (click)="toggleColorWindowPanel()">
+                <mat-icon [svgIcon]="'contrast-box'"></mat-icon>
+            </button>
+            <div *ngIf="colorWindowPanelActive" id="color-window-panel" class="color-window-panel">
+                <mat-card class="color-window-panel--card">
+                    <mat-card-header class="color-window-panel--card__header">
+                        <mat-card-title>Image Window</mat-card-title>
+                        <mat-card-subtitle>Correct image brightness and contrast by tuning parameters below</mat-card-subtitle>
+                        <button mat-icon-button (click)="toggleColorWindowPanel()">
+                            <mat-icon>close</mat-icon>
+                        </button>
+                    </mat-card-header>
+                    <mat-card-content class="color-window-panel--card__content">
+                        <mat-divider [inset]="true"></mat-divider>
+                        <span>
+                            Window width: {{colorWindowWidth}} 
+                        </span>
+                        <mat-slider (change)="changeImageWindowWidth($event.value)"
```

Comment:
Hmm... From the developer's point of view, range of <0, 255> is cool and I'm fine with that. However, why people (especially doctors or medicine students) would like to manipulate with a scan using such a range? Isn't it better to work with floating point value between 0.0 - 100.0? I feel that it would be more natural to them. Don't you think?

It would be the best to use HU values here but it would require a lot of changes both in backend and frontend 😔 Let's keep it in our minds for future 😉

Reply:
As said previously, range is strongly connected with image. From various sources there are claims that numerical values are just for reference (strict values are connected with detailed DICOM files). To sum it up: I don't know 😛, lets find it out using empirical way AKA giving tool to the doctors and studs for usability testing :)

Reply:
Cool! I'm fine with this approach 😉